pkgs/development/python-modules/mkdocs-git-committers-plugin-2/default.nix

```diff
@@ -0,0 +1,43 @@
+{
+  # Evaluation
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pythonAtLeast,
+
+  # Dependencies
+  gitpython,
+  mkdocs,
+  requests,
+}:
+
+buildPythonPackage rec {
+  pname = "mkdocs-git-committers-plugin-2";
+  version = "2.4.1";
+  format = "setuptools";
+
+  disabled = !pythonAtLeast "3.8";
+
+  src = fetchFromGitHub {
+    owner = "ojacques";
+    repo = "mkdocs-git-committers-plugin-2";
+    rev = "refs/tags/${version}";
+    hash = "sha256-hKt0K5gOkdUDwTlyMTwodl4gD1RD+s+CM+zEpngSnlc=";
+  };
+
+  propagatedBuildInputs = [
+    gitpython
+    mkdocs
+    requests
+  ];
+
+  pythonImportsCheck = [ "mkdocs_git_committers_plugin_2" ];
+
```

Comment:
Could you enable tests?

> Make sure the tests are enabled using for example [pytestCheckHook](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pytestcheckhook) and, in the case of libraries, are passing for all interpreters. If certain tests fail they can be disabled individually. Try to avoid disabling the tests altogether. In any case, when you disable tests, leave a comment explaining why.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines-contributing-guidelines

Reply:
Hmmm, the upstream package does not have any test suite, and since it is a plugin that has to do with Git history, mocking one would be pretty complicated I feel. I explicitly disabled the check for now, but if there's any better way to do it, please let me know!